docs/TROUBLESHOOTING.md

```diff
@@ -214,13 +214,19 @@ This section includes how to debug a failed restore. For more specific issues re
 
 <hr style="height:1px;border:none;color:#333;"> 
 
-<h3 align="center">Issue with Backup/Restore of DeploymentConfig using Restic or restore hooks<a id="deployconfig"></a></h3>
+<h3 align="center">Issue with Backup/Restore of DeploymentConfig with volumes or restore hooks<a id="deployconfig"></a></h3>
```

Comment:
```suggestion
<h3 align="center">Issue with Backup/Restore of DeploymentConfig with volumes via file system backup or restore hooks<a id="deployconfig"></a></h3>
```

Reply:
@kaovilai It's not specific to fs backup anymore -- that's why I suggested removing the restic/kopia reference. This is now required for any DCs with volumes, since datamover also needs the pods restored.